src/sidebar/components/test/annotation-omega-test.js

```diff
@@ -72,6 +87,8 @@ describe('AnnotationOmega', () => {
     $imports.$restore();
   });
 
+  it('should test `isSaving`');
```

Comment:
The way `isSaving` works needs some additional scrutiny beyond this PR's focus, so I'm going to leave a note-to-self to add tests in follow-on work.